helm/prometheus-meta-operator/templates/prometheus-rules/service-level.rules.yml

```diff
@@ -14,13 +14,17 @@ spec:
         description: '{{`Service level burn rate is too high for {{ $labels.service }} service.`}}'
         opsrecipe: service-level-burn-rate-too-high/
       expr: |
+        slo_errors_per_request:ratio_rate1h{service=~"api-server|kubelet"}
```

Comment:
I understand why we do this but I'm not sure I like the idea because this will be a nightmare. Would it not make more sense to opt out by area and/or service instead ?

Reply:
I think it should be easier to opt-in that to opt-out